CONTRIBUTING.md

```diff
@@ -1,82 +1,79 @@
 # Lawnchair Launcher Contributing Guidelines
 
-![](https://raw.githubusercontent.com/validcube/lawnchair/pave-path/.github/lawnchair_round.png)
+<img title="Lawnchair Launcher logo" alt="Lawnchair Launcher logo" src="https://raw.githubusercontent.com/validcube/lawnchair/pave-path/.github/lawnchair_round.png" width="100" data-align="inline">
 
-First and foremost, Welcome to the **Lawnchair Launcher** Contributing Guidelines! This document will contains everything you'll need to contribute to the Lawnchair Launcher project.
+First and foremost, welcome to the **Lawnchair Launcher** Contributing Guidelines! This document will contain everything you'll need to contribute to the `Lawnchair Launcher` project.
```

Comment:
**Lawnchair Launcher** is an application, it should be marked as **Bold** instead.

```suggestion
First and foremost, welcome to the **Lawnchair Launcher** Contributing Guidelines! This document will contain everything you'll need to contribute to the **Lawnchair Launcher** project.
```